src/Networking/NexusMods.Networking.NexusWebApi/ModUpdateService.cs

```diff
@@ -65,18 +67,73 @@ await RunUpdateCheck.UpdateModFilesForOutdatedPages(
     /// <inheritdoc />
     public void NotifyForUpdates()
     {
-        var filesInLibrary = NexusModsLibraryItem
+        // Filter out Read-Only items
+
+        // Note(sewer): Mods can generally be broken down into the following categories:
+        // - Mods installed into a Read-Write CollectionGroup [read-write]
+        //     - e.g. 'My Mods' is a Read-Write CollectionGroup
+        // - Mods in Library but not installed (to any CollectionGroup). [read-write]
+        //     - These are mods downloaded from Nexus Mods or manually added via disk.
+        //     - These can 'only' be installed to Read-Write CollectionGroup(s) like 'My Mods'
+        // - Mods in Collections [read-only]
+        //     - These are the mods installed into a CollectionGroup, they cannot be individually removed
+        //       and their lifetime is bound to the Collection.
+        //     - Mods belonging to a collection get auto installed when the collection is fully downloaded, and cannot
+        //       be 'uninstalled' back into the library.
+        
+        // In this case we're looking for all mods which are [read-write], and
+        // not those which are [read-only].
+        //
+        // Although it is easier to code it, so we look for all mods, and take
+        // away the [read-only] ones, whitelisting [read-write] ones instead
+        // will help us avoid risks.
+        //
+        // It's better to have a false negative (can't update that should be updatable)
+        // than a false positive (can update that shouldn't be updatable); because
+        // the former has no potential negative side effects, while the latter
+        // can potentially cause some level of damage.
```

Comment:
Talked with Design, show updates on all items.
We don't want to miss updates for relevant mods like smapi because it is used in one collection.
We don't have replace that would break stuff, so checking all is fine.

Reply:
Reverted for now